src/test/groovy/net/fabricmc/loom/test/integration/KotlinTest.groovy

```diff
@@ -31,7 +31,6 @@ import spock.lang.Unroll
 
 import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 
-@IgnoreIf({ jvm.java16Compatible }) // Fails on J16 > due to https://youtrack.jetbrains.com/issue/KT-45566
```

Comment:
Also need to remove this:

https://github.com/FabricMC/fabric-loom/blob/dev/0.9/.github/workflows/test-push.yml#L48-L50

and

https://github.com/FabricMC/fabric-loom/blob/dev/0.9/.github/workflows/test-push.yml#L77-L79

Reply:
Done.